client/src/Entity/Report/Report.php

```diff
@@ -750,18 +746,30 @@ public function setHasContacts($value)
         return null;
     }
 
-    /**
-     * @return string|null
-     */
-    public function hasDecisions()
+    public function getSignificantDecisionsMade()
     {
-        if (empty($this->getDecisions()) && !$this->getReasonForNoDecisions()) {
-            return null;
-        }
+        return $this->significantDecisionsMade;
+    }
 
-        return $this->getReasonForNoDecisions() ? 'no' : 'yes';
+    public function setSignificantDecisionsMade($significantDecisionsMade)
+    {
+        $this->significantDecisionsMade = $significantDecisionsMade;
+
+        return $this;
     }
 
+//    /**
```

Comment:
Again - not sure if this code was removed temporarily for testing or needs to be removed permanently?